server/src/main/java/org/elasticsearch/index/analysis/Analysis.java

```diff
@@ -247,15 +247,9 @@ public static CharArraySet getWordSet(Environment env, Settings settings, String
         }
     }
 
-    public static List<String> loadWordList(Reader reader, String comment) throws IOException {
+    private static List<String> loadWordList(Path path, String comment) throws IOException {
```

Comment:
I don't think this method should be public. If not, we can create the reader inside.